samtranslator/model/sqs.py

```diff
@@ -34,3 +34,19 @@ def sns_topic_send_message_role_policy(cls, topic_arn, queue_arn):
             ],
         }
         return document
+
+    @classmethod
```

Comment:
Why `@classmethod` instead of `@staticmethod`?

Reply:
Just a copy paste from the method above, I will make it static